### DISCUSSION TITLE
Add button on Image block

### DISCUSSION THREAD

**Discussion by Abhisheknanda1344463** [GrapesJS/grapesjs#3762] [Q&A]
Hi @artf
How can i add button on image block like shown in the picture below.
and i think it's a great idea to implement the stretch button on the image block.
Thanks.

<img width="548" alt="Screenshot 2021-09-10 at 8 03 18 AM" src="https://user-images.githubusercontent.com/20657737/132790108-f5a65189-0a3f-49c4-9481-38362a2d5839.png">

**Comment by Abhisheknanda1344463** [comment]
Anyone can help me on this?

**Reply by artf** [reply]
https://grapesjs.com/docs/modules/Components.html#view
